coriolis/osmorphing/windows.py

```diff
@@ -68,6 +69,39 @@
 REQUIRED_DETECTED_WINDOWS_OS_FIELDS = [
     "version_number", "edition_id", "installation_type", "product_name"]
 
+INTERFACES_PATH_FORMAT = (
+    "HKLM:\\%s\\ControlSet001\\Services\\Tcpip\\Parameters\\Interfaces")
+
+STATIC_IP_SCRIPT_TEMPLATE = """
+$NICS_INFO = ConvertFrom-Json "%(nics_info)s"
+$IPS_INFO = ConvertFrom-Json "%(ips_info)s"
+
+$i = 0
+foreach ($nic in $NICS_INFO) {
+    $mac_address = $nic.mac_address.replace(':', '-').toUpper()
+    $interface = Get-NetAdapter | Where MacAddress -eq $mac_address
+    if (-Not $interface) {
+        $interface = @(Get-NetAdapter | Where Status -eq Up)[$i]
+    }
+    $i++
+
+    Remove-NetIPAddress -InterfaceIndex $interface.ifIndex -Confirm:$false
```

Comment:
Please apply the script structuring standards specific to powershell (i.e. split the above and the below logic within their own well-defined functions, and put the "main() logic" (i.e. the JSON decodes and calls to said functions) at the bottom.)